heapster-saw/doc/tutorial/tutorial.md

```diff
@@ -792,15 +792,12 @@ You will have to generate the `.vo` again to write proofs about
 `incr_ptr`. After you do so, we can easily prove that `incr_ptr`
 produces no errors.
 
-```
-Lemma no_errors_incr_ptr
-  : refinesFun incr_ptr (fun _ => noErrorsSpec).
-Proof.
-  unfold incr_ptr, incr_ptr__tuple_fun.
-  prove_refinement.
-Qed.
 ```
 
+Lemma no_errors_incr_ptr (x: bitvector 64) :
+  spec_refines_eq (incr_ptr x) (safety_spec x).
+Proof.  solve_trivial_spec 0 0. Qed.```
+
```

Comment:
I think this code block is missing a closing ` ``` `.

Reply:
Fixed in latest commit.